Gems/WhiteBox/Code/Source/EditorWhiteBoxComponent.cpp

```diff
@@ -309,7 +309,7 @@ namespace WhiteBox
 
         AZ::Transform worldFromLocal = AZ::Transform::CreateIdentity();
         AZ::TransformBus::EventResult(worldFromLocal, entityId, &AZ::TransformBus::Events::GetWorldTM);
-        m_worldFromLocal = AzToolsFramework::TransformUniformScale(worldFromLocal);
+        m_worldFromLocal = worldFromLocal;
```

Comment:
[minor] This makes an unnecessary copy now. It could just return the result to `m_worldFromLocal`.

````
AZ::TransformBus::EventResult(m_worldFromLocal , entityId, &AZ::TransformBus::Events::GetWorldTM);
````